README.md

```diff
@@ -40,199 +38,107 @@ If you need more information about setting up your development environment Subst
-through Docker.
-
-### Local Development Build
+on each side of the bridge (source and target chain).
 
-#### Building
+### Building
 
-First you'll need to build the bridge node and relay. This can be done as follows:
+First you'll need to build the bridge nodes and relay. This can be done as follows:
 
 ```bash
 # In `parity-bridges-common` folder
 cargo build -p rialto-bridge-node
-cargo build -p ethereum-poa-relay
+cargo build -p millau-bridge-node
+cargo build -p substrate-relay
 ```
 
-Next you'll need to clone the following [fork of OpenEthereum](https://github.com/paritytech/openethereum).
-If you're doing development which only involves the Ethereum to Substrate side of the bridge you may
-use the `master` branch. Otherwise you'll need to checkout the `substrate-builtins-stubs` branch.
-
-```bash
-# Should be at the same level as `parity-bridges-common` folder
-git clone https://github.com/paritytech/openethereum.git openethereum
-git fetch
-git checkout substrate-builtins-stubs
-```
-
-If you've checked out the `substrate-builtins-stubs` branch make sure you've cloned the OpenEthereum
-repo at the same level as `parity-bridges-common` since it references the repo.
-
-Next you'll need to build the Ethereum node:
-
-```bash
-# In `openethereum` folder
-cargo build
-```
-
-#### Running
+### Running
 
 To run a simple dev network you'll can use the scripts located in
-[the `scripts` folder](./scripts). Since the relay connects to both the Substrate and Ethereum
-chains it must be run last.
+[the `scripts` folder](./scripts). Since the relay connects to both Substrate chains it must be run
+last.
 
 ```bash
 # In `parity-bridges-common` folder
-./scripts/run-openethereum-node.sh
 ./scripts/run-rialto-bridge-node.sh
-./scripts/run-eth2sub-relay.sh
+./scripts/run-millau-bridge-node.sh
+./scripts/run-millau-to-rialto-relay.sh
 ```
-At this point you should see the relayer submitting blocks from the Ethereum chain
-to the Substrate chain.
+
+At this point you should see the relayer submitting blocks from the Millau Substrate chain to the
+Rialto Substrate chain.
 
 ### Local Docker Build
 If you want to make a Docker container using your local source files you can run the following
 command at the top level of the repository:
 
 ```bash
-docker build . -t bridge-relay-dev
-```
-
-You can also build and run the Substrate based node as follows:
-
-```bash
-docker build . -t bridge-node-dev --build-arg PROJECT=rialto-bridge-node
+docker build . -t local/rialto-bridge-node --build-arg PROJECT=rialto-bridge-node
+docker build . -t local/millau-bridge-node --build-arg PROJECT=millau-bridge-node
+docker build . -t local/substrate-relay --build-arg PROJECT=substrate-relay
 ```
 
-To run the Substrate node you can do the following:
+You can then run the network as follows:
 
 ```bash
-docker run -it bridge-node-dev --dev --tmp
+docker run -it local/rialto-bridge-node --dev --tmp
```

Comment:
These commands don't quite work at written, but I'll fix it up in the next few days

Reply:
This will be fixed in a follow up PR